tests/components/device_tracker/test_owntracks.py

```diff
@@ -207,20 +208,60 @@
 }
 
 
-class TestDeviceTrackerOwnTracks(unittest.TestCase):
+class BaseMQTT(unittest.TestCase):
```

Comment:
I was under the impression that all test classes should start with `Test` to be discovered